lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -652,18 +652,19 @@ def flatten_array(flattened, array, prefix)
         end
       end
 
-      def headers(options = {})
-        key = options[:key] || @api_key
-        idempotency_key = options[:idempotency_key]
+      def key(options = {})
+        options[:key] || @api_key
```

Comment:
just to double check, do we know that the `@fee_refund_api_key` (which gets set as options[:key]) follows the same format `sk_*` as a regular api key?

Reply:
I'm not sure

Reply:
But they key we use for Authentication for all request would be `options[:key] || @api_key`

Reply:
It also looks like support for the refund_api_key was removed at some point. It's missing all the methods it was created with, https://github.com/activemerchant/active_merchant/commit/36406508acaab8a70c9f1d1866c378a9152f6241

Reply:
i saw [this reference here](https://github.com/activemerchant/active_merchant/blob/4f96d540d676369e2ec2071af3b4f14109e6bd62/lib/active_merchant/billing/gateways/stripe.rb#L169-L174) but not really seeing where we use it on our side of the house so 🤔

Reply:
Yeah I'm not sure either, I think the important key is the one in the headers method. Let me look at the Stripe docs though

Reply:
So we might end up using it because it gets set as key in options which is what we check in the `key?` method. However it looks like it should follow the same `sk...` format if you look at the stripe docs, https://stripe.com/docs/api/fee_refunds/create. However, I don't think we used it cause it's not a field we offer.

Reply:
cool, thanks for checking that!

Reply:
I think it's something that we might need to look at removing later.